var/spack/repos/builtin/packages/ninja/package.py

```diff
@@ -14,6 +14,8 @@ class Ninja(Package):
     url      = "https://github.com/ninja-build/ninja/archive/v1.7.2.tar.gz"
     git      = "https://github.com/ninja-build/ninja.git"
 
+    tags = ['build-tools']
+
     executables = ['^ninja$']
 
     tags = ['e4s']
```

Comment:
`e4s` is going to override the build-tools tag -- and is ninja an official E4S package or just a dependency? @eugeneswalker

Reply:
@sethrj Good catch, this should have been added in the list, I didn't notice the package had already a tag attribute.